src/openlcb/SimpleInfoProtocol.hxx

```diff
@@ -396,23 +396,21 @@ private:
     }
 
     /** Configuration option. See constructor. */
-    unsigned maxBytesPerMessage_ : 8;
+    uint8_t maxBytesPerMessage_;
     /** Configuration option. See constructor. */
-    unsigned useContinueBits_ : 1;
+    uint8_t useContinueBits_ : 1;
```

Comment:
would using bool type be better for this single bit usage?

Reply:
bool is 32 bits.

Reply:
ahh, good point. that is also likely platform specific so best to keep as uint8_t, perhaps drop the single bit usage and keep as uint8_t in that case?

Reply:
a single bit unsigned is perfectly fine for this use case (that's what it used to be before this PR).
I don't see a reason to change.

Reply:
that works for me.